src/raftstore/store/fsm/peer.rs

```diff
@@ -728,6 +752,7 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
                     res.merged,
                     &res.metrics,
                 );
+                self.register_pd_heartbeat_tick();
```

Comment:
Could you add some comments about why we need to heartbeat pd after applying something?